src/test/java/com/uc_it4045/assignment_tracker/AssignmentTrackerApplicationTests.java

```diff
@@ -9,7 +9,7 @@
 class AssignmentTrackerApplicationTests {
 
 	@Autowired
-	private AssignmentTrackerController controller;
+	public AssignmentTrackerController controller;
```

Comment:
fixed variables so that this class can be called correctly throughout the application